spec/requests/list_events_belonging_to_group_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+describe 'GET /groups/:id/events' do
+  let!(:group) { create(:group) } 
+  let!(:events) { 3.times { create(:event) } }
```

Comment:
These events need to belong to the group you created above. You should also create a few events belonging to another group

Reply:
Done